.gitignore

```diff
@@ -124,6 +124,7 @@ bower_components
 
 # Documentation build files.
 doc/build
```

Comment:
@jmchilton You mentioned in https://github.com/galaxyproject/galaxy/pull/2936#issuecomment-248586471 that you put `schema.md` in `.gitignore`, but I can't see it here.